src/test/java/com/salesforce/storm/spout/dynamic/retry/DefaultRetryManagerTest.java

```diff
@@ -539,8 +539,10 @@ public void testNextFailedMessageToRetryWithExpiring() {
 
         // Now advance time by exactly expectedMinRetryTimeMs milliseconds
         retryManager.setClock(
-            Clock.fixed(Instant.ofEpochMilli(FIXED_TIME + (long) (expectedMinRetryTimeMs * expectedDelayMultiplier)), ZoneId.of("UTC"))
-        );
+            Clock.fixed(
+            Instant.ofEpochMilli(FIXED_TIME + (long) (expectedMinRetryTimeMs * Math.pow(expectedDelayMultiplier, 0))),
```

Comment:
This line and the next one should be indented.

Reply:
the checkstyles didn't like the indention for some reason.

Reply:
That's odd! Roger that though, thanks for the explanation.